pkg/apiserver/experiment/experiment.go

```diff
@@ -210,6 +222,12 @@ func (s *Service) createExperiment(c *gin.Context) {
 			_ = c.Error(utils.ErrInternalServer.WrapWithNoMessage(err))
 			return
 		}
+	case "NetworkChaos":
+		if err := s.createNetworkChaos(exp); err != nil {
+			c.Status(http.StatusInternalServerError)
+			_ = c.Error(utils.ErrInternalServer.WrapWithNoMessage(err))
```

Comment:
IMO we can add some info to show the chaos kind, which is better than just returning error